tutorials/intermediate/arranging_axes.py

```diff
@@ -0,0 +1,371 @@
+"""
+=====================================
+Arranging multiple *Axes* in a Figure
+=====================================
+
+Often more than one *Axes* is wanted on a figure at a time, usually
+organized into a regular grid.  Matplotlib has a variety of tools for
+working with grids of *Axes* that have evolved over the history of the library.
+Here we will discuss the tools we think users should use most often, the tools
+that underpin how *Axes* are organized, and mention some of the older tools.
+
+.. note::
+
+    Matplotlib uses *Axes* to refer to an *Artist* that contains plotting
+    *Artists* and the x- and y-*Axis*.  Another term that is often used is
```

Comment:
Can we make that more informal? Introducing *Artist* as another technical term 

In Matplotlib, an *Axes* is the container object for one data drawing area including, x- and y-axis, ticks, labels, title, etc. See `<figure_parts>`_ for more details.